src/UsgsAstroSarSensorModel.cpp

```diff
@@ -355,6 +355,42 @@ csm::ImageCoord UsgsAstroSarSensorModel::groundToImage(
   }
 }
 
+csm::ImageCoord UsgsAstroSarSensorModel::groundToImage(
+    const csm::EcefCoord& groundPt,
+    const std::vector<double> adj,
+    double desiredPrecision,
+    double* achievedPrecision,
+    csm::WarningList* warnings) const {
+
+  // Find time of closest approach to groundPt and the corresponding slant range by finding
+  // the root of the doppler shift frequency
+  try {
+    double timeTolerance = m_exposureDuration * desiredPrecision / 2.0;
+
+    // Adjust the ground point value (I think. Check notes... maybe not) 
+    csm::EcefCoord adjGroundPt(groundPt.x + getValue(0, adj), groundPt.y + getValue(1, adj), 
+                                   groundPt.z+ getValue(2, adj));
+    double time = dopplerShift(adjGroundPt, timeTolerance);
+    double slantRangeValue = slantRange(groundPt, time);
```

Comment:
The adjustment vector should be passed to these functions because they get the sensor position and velocity.